src/Mantasflowers.WebApi/Setup/Database/DatabaseSetup.cs

```diff
@@ -1,4 +1,5 @@
 using Mantasflowers.Persistence;
+using Mantasflowers.WebApi.Extensions;
```

Comment:
?

Reply:
Nothing logic wise changed, just moved Configuration extension for reuse.